operator/ARCHITECTURE.md

```
@@ -0,0 +1,235 @@
+### Cross-CR interaction
+
+Central and SecuredCluster reconcilers watch each other's CR type. This enables:
+
+- **Central** to auto-generate init bundles when a SecuredCluster appears in
+  the same namespace.
+- **SecuredCluster** to decide whether to deploy a local scanner based on
+  whether a Central exists in the same namespace.
+
+## Directory layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)